src/templates/indexPage/index.js

```diff
@@ -104,24 +104,15 @@ const IndexPageTemplate = ({ language, data }) => {
         <p className="bodyLarge">{investorPortal.text}</p>
         <Button to={`/${strippedLanguage}/products`} text="Learn more" />
         <div className={styles.investorPortalImagesWrapper}>
-          <ImageWrapper
-            src={investorPortal.image}
-            alt="Abstract investorportal"
-            sizes={findImageSizes(investorPortal.image, imageSizes)}
-            outerWrapperClassName={styles.imageContainer}
-          />
-          <ImageWrapper
-            src={investorPortal.image}
-            alt="Abstract investorportal"
-            sizes={findImageSizes(investorPortal.image, imageSizes)}
-            outerWrapperClassName={styles.imageContainer}
-          />
-          <ImageWrapper
-            src={investorPortal.image}
-            alt="Abstract investorportal"
-            sizes={findImageSizes(investorPortal.image, imageSizes)}
-            outerWrapperClassName={styles.imageContainer}
-          />
+          {investorPortal.investorPortalImages.map(investorPortalImage => (
+            <ImageWrapper
+              key={investorPortalImage.name}
+              src={investorPortalImage.image}
+              alt={investorPortalImage.name}
+              sizes={findImageSizes(investorPortalImage.image, imageSizes)}
+              outerWrapperClassName={styles.imageContainer}
+            />
+          ))}
```

Comment:
To ting i Firefox:

1. Er det ikke naturlig at scrollbarene ikke synes på desktop?
2. Burde ikke dette være ett scroll-område istedenfor to som scroller hver for seg?

![image](https://user-images.githubusercontent.com/5845924/43715111-e0612ccc-996e-11e8-9437-a8e5cb93654c.png)

Reply:
Jeg ser de bare når jeg scroller i min firefox?
Kanskje det burde vært bare 1? dunno. Men det ligger allerede på master
Er ikke det jeg har endret på.